Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

```diff
@@ -2971,3 +2971,9 @@ Creates a process search job and returns results if 'polling' argument is True.
 | CarbonBlackEEDR.SearchProcess.results.process_username | String | User names related to the process. | 
 | CarbonBlackEEDR.SearchProcess.results.regmod_count | Number | The cumulative count of registry modifications since process tracking started. | 
 | CarbonBlackEEDR.SearchProcess.results.scriptload_count | Number | The cumulative count of loaded scripts since process tracking started. | 
+
+
+## Troubleshooting
+
+Note, If the API returns duplicates, we will drop them, causing the total number of alerts fetched to potentially be fewer than the set limit.
```

Comment:
```suggestion
Note: If the API returns duplicates, the duplicates will be dropped, causing the total number of alerts fetched to potentially be fewer than the set limit.
```
This is really not a troubleshooting issue. Not sure it belongs under a Troubleshooting heading.  It's just a Note.  Perhaps it belongs with **fetch-incidents**.